bin/compile

```diff
@@ -6,32 +6,31 @@ $stdout.sync = true
 BUILD_DIR = ARGV[0]
 CACHE_DIR = ARGV[1]
 ARCHIVE_NAME = "freetds-patched"
+INSTALL_DIR = "#{BUILD_DIR}/vendor/freetds"
 
 puts "-----> FreeTDS building in #{BUILD_DIR}"
+puts `mkdir -p #{INSTALL_DIR}`
 
 puts "-----> Fetching & Extracting FreeTDS"
-puts `mkdir -p #{CACHE_DIR}`
-puts `curl ftp://ftp.freetds.org/pub/freetds/stable/#{ARCHIVE_NAME}.tar.gz -o - | tar -xz -C #{CACHE_DIR} -f -`
+puts `curl ftp://ftp.freetds.org/pub/freetds/stable/#{ARCHIVE_NAME}.tar.gz -o - | tar -xz -C #{INSTALL_DIR} -f -`
 
-TARGET_DIR = `ls #{CACHE_DIR} | grep 'freetds-[0-9]\.'`.chomp
+TARGET_DIR = `ls #{INSTALL_DIR} | grep 'freetds-[0-9]\.'`.chomp
 puts "installing #{TARGET_DIR}"
 
-puts "-----> Configurating FreeTDS"
-puts `mkdir -p #{BUILD_DIR}/vendor/freetds`
-puts `cd #{CACHE_DIR}/#{TARGET_DIR} && ./configure --prefix=#{BUILD_DIR}/vendor/freetds --with-libiconv-prefix=DIR --with-openssl=DIR --with-tdsver=7.3`
```

Comment:
I *think* the `--prefix` option means that CACHE_DIR has all the source files and then the "made" (compiled/runtime) files are spit out into BUILD_DIR, so what eventually ends up in app/vendor/free-tds is not the full source.

In terms of keeping unnecessary files outside of the final slug, I think the best solution would be to:
1. Download the files into the BUILD_DIR
2. Specify a prefix of `/app/vender/freetds`
3. `make` and `make install` to write the compiled files into the correct directory.